test/utils/utils.ts

```diff
@@ -97,3 +97,17 @@ export const submitProposal = async (
   );
   return proposalCreatedEvent.args[0]; // Returns the proposalId
 };
+
+export const unpauseMentoToken = async (
```

Comment:
If we are not reusing this across files maybe we can just put it under the test file itself?

Reply:
Yeah initially I thought it might be useful for other tests as well but so far we don't need the token to be unpaused anywhere else 👍